starter-code/src/index.js

```diff
@@ -1,8 +1,58 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from "./App.js";
+// import App from "./App.js";
+import "./App.css";
 
 ReactDOM.render(
-  <App />,
+
+  <section className="whole-wrap">
```

Comment:
with react, after setting up the index.js file, we don't add much code in here . So here you would have 
```
ReactDOM.render(
  <App />,
  document.getElementById("root")
)
```
with the rest of the code going in App.js

over the next few lessons you'll understand why we do this as well as the importance of keeping the code well structured